doc/sphinxman/source/build_planning.rst

```diff
@@ -194,7 +194,7 @@ that software for |PSIfour| and any notes and warnings pertaining to it.
 
 * NumPy (needed at runtime *and* buildtime) http://www.numpy.org/
 
-* mpmath (needed at buildtime) http://mpmath.org/
+* mpmath (only needed if you build gau2grid to angular momentum >16) http://mpmath.org/
```

Comment:
Seems like a reasonable requirement.  AM=16 for a theory with no electron-electron cusps should be more than sufficient, methinks!

Reply:
Yup, easy enough to serialize your RSH code. No need for another dependancy.